modules/internal/ChapelBase.chpl

```diff
@@ -3441,4 +3441,30 @@ module ChapelBase {
   inline proc chpl_field_gt(a, b) where !isArrayType(a.type) {
     return a > b;
   }
+
+  export inline proc chpl_sqrt64(x: real(64)): real(64) {
```

Comment:
I would expect these to be in `runtime/include/chplmath.h` and just be called from the code generator. AFAIK you don't even have to declare them as `extern proc` in the Chapel code at all.

Reply:
Done in https://github.com/chapel-lang/chapel/pull/24455/commits/bec6eeac42644a39cb1940a23cae240c8cdc9d8f.